### PR TITLE
Use background-size only when hi-res image is used.

### PR DESCRIPTION
**Pull request by Holdwyn**
fixes issue #9653 
background-size was added while implementing support for high res images (to avoid clipped images?), so I guess it is safe to only add the exact size when a high res source is found.

### CODE REVIEW

**Review by johnspackman — DISMISSED** [review]

**Review by oetiker — DISMISSED** [review]

File: framework/source/class/qx/ui/table/cellrenderer/AbstractImage.js [hunk 1]
```diff
@@ -232,15 +232,15 @@ qx.Class.define("qx.ui.table.cellrenderer.AbstractImage",
     _getContentHtml : function(cellInfo)
     {
       var content = "<div></div>";
-
-      // background size is critical for high-resolution images
-      var backgroundSize = this.__imageData.width + "px " + this.__imageData.height + "px";
+      var backgroundSize = "auto auto";
```
Comment (hkollmann) [inline framework/source/class/qx/ui/table/cellrenderer/AbstractImage.js]: The original code did not use background-size.

See here: https://github.com/qooxdoo/qooxdoo/commit/0937f6c9ef8bf14ca05a9f778001b8d9290b0804#diff-8fdc282bf7f5c55d884a72d38bd81948

**Comment by hkollmann** [comment]
Second idea: The problem only occurs with combined icons. What's about checking if it's a combined image with qx.util.ResourceManager.getInstance().getCombinedFormat(this.__imageData.url) === "" and using background-size only in this case?

File: framework/source/class/qx/ui/table/cellrenderer/AbstractImage.js (hunk 1 above)
Comment (Holdwyn) [reply framework/source/class/qx/ui/table/cellrenderer/AbstractImage.js]: Yes but background-size: auto auto; is the default value. The code would become more complicated building two different style objects (one with and one without background-size).

**Comment by Holdwyn** [comment]
> Second idea: The problem only occurs with combined icons. What's about checking if it's a combined image with qx.util.ResourceManager.getInstance().getCombinedFormat(this.__imageData.url) === "" and using background-size only in this case?

Does this work with the url of the image? The function has resource id as parameter.

**Comment by hkollmann** [comment]
Is the same id as used in findHighResolutionSource: http://www.qooxdoo.org/current/api/#qx.util.ResourceManager

File: framework/source/class/qx/ui/table/cellrenderer/AbstractImage.js (hunk 1 above)
Comment (hkollmann) [reply framework/source/class/qx/ui/table/cellrenderer/AbstractImage.js]: OK

**Comment by hkollmann** [comment]
Why did you close this PR?

**Comment by Holdwyn** [comment]
> Why did you close this PR?

by mistake :blush:

**Review by hkollmann — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

**Review by level420 — APPROVED** [review]

**Review by derrell — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Pull request merged**